test_scripts/API/VehicleData/SubscribeVehicleData/003_RPC_parameter_already_subscribed_Result_IGNORED_flow.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Item: Use Case: request is allowed by policies but app is already subscribed for specified parameter
+--
+-- Requirement summary:
+-- [SubscribeVehicleData] As a mobile app wants to send a request to subscribe for specified parameter
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid SubscribeVehicleData to SDL and this request
+--    is allowed by Policies but app is already subscribed for specified parameter
+-- SDL must:
+-- 1) SDL responds IGNORED, success:true and info: "Already subscribed on some provided VehicleData."
```

Comment:
@AByzhynar, should be success:false, according to the requirements.

Reply:
@AStasiuk Fixed in 3b703abdab25b3e13b150954e8a8c707f40a3b45